app/src/androidTest/java/it/niedermann/nextcloud/deck/util/ColorUtilTest.java

```diff
@@ -96,4 +98,25 @@ public void testContrastRatioIsSufficient() {
             );
         }
     }
+
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void testGetCleanHexaColorString() {
+        final List<Pair<String, String>> validColors = new ArrayList<>();
+        validColors.add(new Pair<>("#0082C9", "#0082C9"));
+        validColors.add(new Pair<>("0082C9", "#0082C9"));
+        validColors.add(new Pair<>("#CCC", "#CCCCCC"));
+        validColors.add(new Pair<>("ccc", "#cccccc"));
```

Comment:
Please consider to test `#af0` value, with a correct result as `#aaff00`.

Reply:
You are absolutely right, i will fix this asap. Jt was supposed to be a quickfix to avoid the crash.

Maybe you want to contribute this method? :rocket:

Reply:
Sorry bro, I'm not familiar with Java or Kotlin 🙃

Reply:
In Swift it might be like this:
```swift
input.reduce("#") { result, char in
  let char = String(char)
  return result.appending(char + char)
}
```

Reply:
No problem, @desperateCoder is already on it! :)

Reply:
i shudder to think about iOS. wrong app, bro :laughing:

Reply:
@m4rr i added a test case, hope this fixes pretty much anything parsable: https://github.com/stefan-niedermann/nextcloud-deck/commit/87bce6d14086b309c0dbcf2502c4c78cdb4147c1